scripts/setup-adapters.sh

```diff
@@ -22,13 +22,21 @@ set -eufx -o pipefail
 SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 source $SCRIPTDIR/setup-helper-functions.sh
 DEPENDENCY_DIR=${DEPENDENCY_DIR:-$(pwd)}
+CMAKE_BUILD_TYPE="${BUILD_TYPE:-Release}"
 
-function install_aws-sdk-cpp {
+function install_aws_deps {
   local AWS_REPO_NAME="aws/aws-sdk-cpp"
   local AWS_SDK_VERSION="1.11.169"
 
   github_checkout $AWS_REPO_NAME $AWS_SDK_VERSION --depth 1 --recurse-submodules
-  cmake_install -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management"
+  cmake_install -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE} -DBUILD_SHARED_LIBS:BOOL=OFF -DMINIMIZE_SIZE:BOOL=ON -DENABLE_TESTING:BOOL=OFF -DBUILD_ONLY:STRING="s3;identity-management"
+  # Dependencies for S3 testing
+  if [[ "$OSTYPE" == linux-gnu* ]]; then
```

Comment:
Curious why this is required - since we are building a centos 8 container anyway. It doesnt look like we support any other OS (no separate check for MacOs etc..)

Reply:
This setup-adapters.sh script is used by MacOS users too. I can add instructions for MacOS. You will see checks for MacOS elsewhere in the file.